### ISSUE TITLE
Copying a notebook in a subdirectory leads to a "No such file or directory error".

### ISSUE THREAD

**Issue by akshayka** [marimo-team/marimo#2307]
### Describe the bug

Directory structure:

```
folder/
  hello_world.py  
```

1. Run `marimo edit` in root (outside `/folder`)
2. Open `folder/hello_world.py`
3. Create notebook copy with notebook action

Yields

```
{"detail":"[Errno 2] No such file or directory: '/home/akshay/tmp/_hello_world.py'"}
```

in a new tab. Upon inspection the file was successfully copied as `folder/_hello_world.py` but marimo tried opening it in root instead of `folder/` .

cc @wasimsandhu

### Environment

0.8.14

**Comment by wasimsandhu** [comment]
Checking it out now, thanks

**Comment by akshayka** [comment]
Fixed by #2308
